client/http/types.go

```diff
@@ -34,6 +34,15 @@ type ClusterState struct {
 	ReplicationStatus string    `json:"replication_status"`
 }
 
+// State is the status of PD server.
+// NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
```

Comment:
Maybe it would be better to associate to the original type place?

Reply:
done